src/components/card/CardImage.js

```diff
@@ -35,29 +35,32 @@ export default class CardImage extends BaseComponent {
      */
     position: PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)]),
     /**
-     * border radius, basically for Android since overflow doesn't work well
+     * border radius, basically for Android since overflow doesn't work well (deprecated)
      */
     borderRadius: PropTypes.number
   };
 
-  static defaultProps = {
-    borderRadius: BorderRadiuses.br40
-  };
+  constructor(props) {
+    super(props);
+
+    if (props.borderRadius) {
+      LogService.warn('uilib: Please stop passing borderRadius to Card.Image, it will get the borderRadius from the Card');
+    }
```

Comment:
I don't think people pass borderRadius to CardImage. 
I think it's an internal prop, we injected it in the Card code.

Reply:
Unfortunately they do, I've seen usages, but we might be able to ignore because the two usages I've seen pass the same radius as the card (I did not look for more usages).
What would you like me to do?

Reply:
Let's ignore them. 
But still keep the warning so people remove it.

Reply:
I see u already do that. so nvm (: